components/datetime/tests/patterns/tests/timezones.json

```diff
@@ -0,0 +1,405 @@
+[
```

Comment:
Just leaving a comment because my GitHub collapses this file by default, and I'm hoping this will make it open.